### DISCUSSION TITLE
[INTERAÇÕES-INTEGRAÇÕES-INFORMAÇÕES]  DISCORD PIX BRASIL.

### DISCUSSION THREAD

**Discussion by PIX-DISCORD** [bacen/pix-api#234] [Show and tell] [answered]
Informo que temos o nosso canal de comunicação rápida, direta e ao vivo entre os integradores, informadores, com interações entre os membros do ecossistema PIX de forma simples, dinâmica, participativa e principalmente colaborativa. Somos uma comunidade pioneira e decidida a fazer o PIX chegar a patamares ainda maiores de usabilidades e sendo uma solução financeira rápida e eficiente. Participe = https://discord.gg/mJGzEEPctr

**Comment by renatofrota** [comment, marked as answer]
Ótima iniciativa. Em conjunto com o [Pix-Pendencias](https://github.com/renatofrota/pix-pendencias/) está sendo muito útil para os PSPs, certamente!

**Reply by PIX-DISCORD** [reply]
Sim queremos integrar a comunidade PIX do Brasil para soluções para todos.
